models/students_account.go

```diff
@@ -0,0 +1,33 @@
+package models
+
+import "github.com/fatih/structs"
+
+type StudentAccount struct {
+	ID                    int     `json:"id" bson:"id" structs:"id" db:"id"`
+	StudentFee            float64 `json:"studentFee" bson:"student_fee" structs:"student_fee" db:"student_fee"`
```

Comment:
these fields should not be part of student ,but rather accounts, create accounts model and accomodate students and teacher in it, think of it like in future if we need tracking for cleaning staff in your approach we would need to create a new model again , his is not an ideal solution. Use one model to accomodate all soltions